integrations/google_vertex/src/haystack_integrations/components/generators/google_vertex/captioner.py

```diff
@@ -12,16 +12,40 @@
 
 @component
 class VertexAIImageCaptioner:
+    """
```

Comment:
copy pasted all this stuff from our docs (e.g. https://docs.haystack.deepset.ai/v2.0/docs/vertexaitextgenerator)